doctrine:
    dbal:
        url: '%env(resolve:DATABASE_URL)%'

        # IMPORTANT: You MUST configure your server version,
        # either here or in the DATABASE_URL env var (see .env file)
        #server_version: '16'

        profiling_collect_backtrace: '%kernel.debug%'
        use_savepoints: true
    orm:
        auto_generate_proxy_classes: true
        enable_lazy_ghost_objects: true
        report_fields_where_declared: true
        validate_xml_mapping: true
        naming_strategy: doctrine.orm.naming_strategy.underscore_number_aware
        auto_mapping: true
        mappings:
            App:
                type: attribute
                is_bundle: false
                dir: '%kernel.project_dir%/src/Entity'
                prefix: 'App\Entity'
                alias: App
        filters:
            soft_delete:
                class: Gedmo\SoftDeleteable\Filter\SoftDeleteableFilter
                enabled: true        
when@prod:
    doctrine:
        orm:
            auto_generate_proxy_classes: false
            proxy_dir: '%kernel.build_dir%/doctrine/orm/Proxies'
            query_cache_driver:
                type: pool
                pool: doctrine.system_cache_pool
            result_cache_driver:
                type: pool
                pool: doctrine.result_cache_pool

    framework:
        cache:
            pools:
                doctrine.result_cache_pool:
                    adapter: cache.app
                doctrine.system_cache_pool:
                    adapter: cache.system




#/var/www/html/projekat3$ php bin/console doctrine:schema:validate

#Mapping
#-------

                                                               
#[OK] The mapping files are correct.                           
                                                               

#Database
#--------


# In ExceptionConverter.php line 101:
#                                                                
#   An exception occurred in the driver: SQLSTATE[HY000] [2002] 
#   php_network_getaddresses: getaddrinfo for symfony-6-testing-project_db failed: Temporary failure in name resolution                                      
#                                                               
# In Exception.php line 28:
#                                                               
#   SQLSTATE[HY000] [2002] php_network_getaddresses: getaddrin  
#   fo for symfony-6-testing-project_db failed: Temporary failure in name resolution                                      
#                                                               
# In Driver.php line 33:
#                                                               
#   SQLSTATE[HY000] [2002] php_network_getaddresses: getaddrin  
#   fo for symfony-6-testing-project_db failed: Temporary failure in name resolution                                      
#                                                               
# In Driver.php line 33:
#                                                               
#   PDO::__construct(): php_network_getaddresses: getaddrinfo   
#   for symfony-6-testing-project_db failed: Temporary failure  
#    in name resolution                                         

#doctrine:schema:validate [--em EM] [--skip-mapping] [--skip-sync] [--skip-property-types]